10-JWT/README.md

```diff
@@ -0,0 +1,220 @@
+# Overview
+
+In the previous Security lab, we focused on perimeter security and configured policies to restrict access based on the Workspace.
+In this lab we will secure the public ingress and enforce user authentication so that only properly authenticated users taht have the
```

Comment:
```suggestion
In this lab we will secure the public ingress and enforce user authentication so that only properly authenticated users that have the
```